tests/unit/tuner/callback/test_early_stopping.py

```diff
@@ -0,0 +1,175 @@
+import numpy as np
+import paddle
+import pytest
+import tensorflow as tf
+import torch
+
+from finetuner.tuner.base import BaseTuner
+from finetuner.tuner.callback import EarlyStopping
+from finetuner.tuner.keras import KerasTuner
+from finetuner.tuner.paddle import PaddleTuner
+from finetuner.tuner.pytorch import PytorchTuner
+from finetuner.tuner.state import TunerState
+
+
+@pytest.fixture(scope='module')
+def pytorch_model() -> BaseTuner:
+    embed_model = torch.nn.Sequential(
```

Comment:
Simplify this so that it is max 1 line - we need the simplest model possible. Same goes for all frameworkds